src/http2_connection.erl

```diff
@@ -835,7 +835,10 @@ route_frame(F={H=#frame_header{stream_id=StreamId}, #window_update{}},
               }}
 
     end;
-
+route_frame({#frame_header{type=T}, _}, Conn)
+  when T > ?CONTINUATION ->
+    lager:debug("Ignoring Unsupported Expansion Frame"),
+    {next_state, connected, Conn};
```

Comment:
Alright, good then. Would probably warrant a little comment if you think so, but yeha.